### PR TITLE
Added the OpenHack link to the README

### PR DESCRIPTION
**Pull request by michelle123lam**
This is my first stab at #1 . Let me know what you think!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
-## [OpenHack]: Your first PR!
+## [OpenHack](http://openhack.info): Your first PR!
```
Comment (bclyang) [inline README.md]: Please link to the HTTPS version

**Review by bclyang — COMMENTED** [review]
Thanks for helping out! Just that one thing :)

File: README.md (hunk 1 above)
Comment (michelle123lam) [reply README.md]: Ok, I made the change.

**Comment by bclyang** [comment]
LGTM!

**Pull request merged**